From 04d7fa77910bd500fb175c5384bb7220cbc3227f Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:45:36 -0400
Subject: [acpi] numa: Fix acpi_get_node() prototype

Message-id: <20140806034535.1438.52341.stgit@gir.stowe>
Patchwork-id: 87423
O-Subject: [RHEL7.1 PATCH 14/69] ACPI / numa: Fix acpi_get_node() prototype
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: 486c79b5002e4a75c1c3614180702196ec6d904d

commit 486c79b5002e4a75c1c3614180702196ec6d904d
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 24 15:25:10 2014 -0700

    ACPI / numa: Fix acpi_get_node() prototype

    acpi_get_node() takes an acpi_handle, not an "acpi_handle *".  This
    fixes the prototype and the definitions.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/numa.c b/drivers/acpi/numa.c
index 09f79a2..acf9ca4 100644
--- a/drivers/acpi/numa.c
+++ b/drivers/acpi/numa.c
@@ -332,7 +332,7 @@ int acpi_get_pxm(acpi_handle h)
  return -1;
 }
 
-int acpi_get_node(acpi_handle *handle)
+int acpi_get_node(acpi_handle handle)
 {
  int pxm, node = NUMA_NO_NODE;
 
diff --git a/include/linux/acpi.h b/include/linux/acpi.h
index e2e52cf..d1d2483 100644
--- a/include/linux/acpi.h
+++ b/include/linux/acpi.h
@@ -239,13 +239,13 @@ extern void acpi_osi_setup(char *str);
 
 #ifdef CONFIG_ACPI_NUMA
 int acpi_get_pxm(acpi_handle handle);
-int acpi_get_node(acpi_handle *handle);
+int acpi_get_node(acpi_handle handle);
 #else
 static inline int acpi_get_pxm(acpi_handle handle)
 {
  return 0;
 }
-static inline int acpi_get_node(acpi_handle *handle)
+static inline int acpi_get_node(acpi_handle handle)
 {
  return 0;
 }
-- 
1.7.1